### DISCUSSION TITLE
[WASM] Splash screen trouble after most recent updates

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#12237] [Q&A] [answered]
It always was the case that there was a funny splash screen flickering - where actually the splash screen was shown twice - on WASM.
However, it was not that bad since the logos shown on the splash screen had the same scaling.

Unfortunately most recent updates (please see screenshot below) broke that: now the 1st splash logo is way larger than the second - which magnifies the underlying problem.

What could I do a make sure there only would be 1 splash logo displayed - or if still not possible - how could I revert to the prior behaviour where the logos of both splash images had the same scaling?

@MartinZikmund I recall I read something related recently where you fixed some splash screen logic which sounded similar to my 
'2 splash screens' problem (couldn't find the link again though). Could that help?

**Comment by MartinZikmund** [comment]
@DierkDroth Can you please add the screenshot? What version of bootstrapper are you using? And what version of Uno?

**Reply by DierkDroth** [reply]
@MartinZikmund sorry forgot the screenshot

<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/56084fbe-1967-4bac-9bd8-e2842048afb1">

**Reply by DierkDroth** [reply]
@MartinZikmund would you have any feedback on the issue above? Thanks in advance

**Reply by MartinZikmund** [reply, marked as answer]
Hi, sorry about the delay! To achieve the same CSS styling, I would suggest replicating the following style:

https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/f335ab6c9c31be2d3db727c09607aaf84b8023b1/src/Uno.Wasm.Bootstrap/WasmCSS/uno-bootstrap.css#LL20C1-L31C6

In the CSS file within the Uno Platform WASM project, but targeting `.uno-splash`. Maybe you will need to reset some of the properties set by this style:

https://github.com/unoplatform/uno/blob/89e5df15c35cd34bd454e8b4256cc07c745a9867/src/Uno.UI/WasmCSS/Uno.UI.css#LL240C1-L252C2

Sorry this is such hassle currently, in the future this will be much simpler (especially if you update to the latest 8.x bootstrapper and latest Uno versions, there will now but just one single splash screen, so things will look much better)

**Reply by DierkDroth** [reply]
Thanks for your feedback @MartinZikmund 

Curious:
- I suspect to use the 8.x boostrapper and latest UNO bits I actually have to upgrade to the .NET 8 preview + VS preview, no?
- I suppose the 8.X stuff (Microsoft and UNO) might not be stable enough yet for some real usage?!?

**Reply by jeromelaban** [reply]
@DierkDroth as usual, the `-dev` bits are in development and may break at any point in time. There is no need to update to any .NET 8 preview for webassembly at this time.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban. Guess I'll wait another 6 months then...
